src/config.js

```diff
@@ -3,37 +3,37 @@
 const chalk = require("chalk")
 
 module.exports = {
-    error: {
+    ERROR: {
```

Comment:
Can you add a test for this change - i.e a test that would have failed before and is now passing